src/frontend/package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "@azavea/echo-locator",
+  "description": "Accessibility analysis for housing",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "webpack --progress --env production --mode=production",
+    "fixlint-js": "eslint --ignore-path src/.eslintignore --fix src",
+    "fixlint-styles": "stylelint '**/*.+(css|scss|sass)' --fix",
+    "format": "prettier --config .prettierrc --write 'src/**/*.{js,jsx}'",
+    "lint": "eslint src --ignore-path src/.eslintignore; stylelint '**/*.+(css|scss|sass)' --config .stylelintrc",
+    "lint-js": "eslint src --ignore-path src/.eslintignore",
+    "lint-styles": "stylelint '**/*.+(css|scss|sass)' --config .stylelintrc",
+    "semantic-release": "semantic-release",
+    "start": "webpack-dev-server --env development --mode development --open",
+    "test": "eslint src --ignore-path src/.eslintignore && stylelint '**/*.+(css|scss|sass)' --config .stylelintrc"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/azavea/echo-locator.git"
+  },
+  "keywords": [
+    "echo-locator",
+    "frontend",
+    "transit"
+  ],
+  "author": "Azavea",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/azavea/echo-locator/issues"
+  },
+  "homepage": "https://github.com/azavea/echo-locator",
+  "dependencies": {
+    "@conveyal/gridualizer": "~2.1.0",
+    "@conveyal/lonlat": "~1.4.0",
+    "@conveyal/woonerf": "~4.2.1",
+    "@mapbox/polyline": "~1.1.0",
+    "axios": "0.21.1",
+    "babel-plugin-css-modules-transform": "~1.6.2",
+    "d3-scale": "~2.2.2",
+    "date-fns": "~1.30.1",
+    "debug": "~4.1.1",
+    "focus-visible": "~5.0.2",
+    "geojson-bbox": "~0.0.0",
+    "i18next": "^21.4.0",
+    "isomorphic-fetch": "~2.2.1",
+    "jsolines": "~2.1.2",
+    "leaflet": "~1.5.1",
+    "lodash": "~4.17.15",
+    "logrocket": "^1.0.6",
+    "logrocket-react": "~4.0.1",
+    "react": "~16.14.0",
```

Comment:
I updated the minor version of `react` and `react-dom` from `16.13.0` to `16.14.0`, because we do not use `import React to "react"` everywhere, and:

>[React 17 RC](https://reactjs.org/blog/2020/08/10/react-v17-rc.html) already includes support for the new transform, so go give it a try! To make it easier to adopt, we’ve also backported its support to React 16.14.0, React 15.7.0, and React 0.14.10. You can find the upgrade instructions for different tools [below](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#how-to-upgrade-to-the-new-jsx-transform).

See more here: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-a-jsx-transform